src/utils/batchRunAll.ts

```diff
@@ -0,0 +1,39 @@
+import * as CliProgress from 'cli-progress';
+
+export async function batchRunAll<I>(

```

Comment:
this isn't done in batches anymore. This is better called runWithConcurrentLimit() or something like that